content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -414,6 +414,17 @@ paketo-buildpacks_datadog/helper/exec.d/toggle': exit status 1
 
 The solution to this issue is to upgrade to version 4.6.0 or later.
 
+##### Problem activating Datadog tracer
+
+You might encounter initialization errors if your tracer configuration relies on Unix Domain Sockets (UDS), which are not supported in native images:
+
+```text
+dd.trace 2024-12-30 08:34:43:306 +0000] [main] WARN datadog.trace.agent.tooling.nativeimage.TracerActivation - Problem activating datadog tracer
+java.lang.NoClassDefFoundError: Could not initialize class jnr.unixsocket.UnixSocketChannel
+```
+
+The solution is to configure the Java tracer to use host-based communication (`hostip` or `service` mode) rather than socket-based communication (`socket` mode). For more information, see [Configure APM and DogstatsD communication mode][11].
```

Comment:
```suggestion
The solution is to configure the Java tracer to use host-based communication (`hostip` or `service` mode), rather than socket-based communication (`socket` mode). For more information, see [Configure APM and DogstatsD communication mode][11].
```